support/docs/configuration.md

```diff
@@ -1,33 +1,48 @@
-Configuration
+Full configuration
 -------------
+Need to know how to use the transport? [take a look here](transport.md).
+The 'how to send a email' [can be found here](usage.md).
```

Comment:
`can be found here` is niet zo handig, je kunt er bijvoorbeeld van maken `see the usage documentation`